package.json

```diff
@@ -39,6 +39,7 @@
 		"vue-observe-visibility": "^0.4.6",
 		"vue-prevent-unload": "^0.2.3",
 		"vue-router": "^3.1.3",
+		"vue-scroll": "^2.1.13",
 		"vue-virtual-scroll-list": "^1.4.4",
```

Comment:
Remove as its not used anymore?

Reply:
Nope, we do use it :) 
https://github.com/nextcloud/spreed/pull/2692/files#diff-1c90ff38a08209f9ebd4d05d1e43358eR68

Reply:
I commented on vue-virtual-scroll-list....